backend/src/controllers/review.controller.ts

```diff
@@ -159,12 +159,18 @@ Consider the further instructions and example replies if provided by the manager
     const wordFrequency = new Map<string, number>();
     this.debug(`Calculating word frequency for ${reviews.length} reviews`);
 
+    const isWantedWord = (word: string) =>
+      !['was', 'the', 'is', 'a', 'to', 'into', 'of', 'at', 'on', 'in', 'are', 'and'].includes(word.toLowerCase());
```

Comment:
Bug: The trending words panel contained mainly words that aren't really meaningful.
Solution: Filter the words and remove all non meaningful words. This is a temporary hardcoded solution